### DISCUSSION TITLE
Custom CSS is not working on Posts

### DISCUSSION THREAD

**Discussion by JonHjartarson** [elementor/elementor#20746] [Developers Community]
Hello there,

Im trying to style the headlines in my post list in Elementor pro with CSS. It works when I add the code in in the custom CSS window in edit mode but when I publish the changes they wont appear on the page. See the screenshots.
![1](https://user-images.githubusercontent.com/120577855/207619967-728fa600-6b39-4477-8528-90a1b83cdff0.jpg)
![2](https://user-images.githubusercontent.com/120577855/207619980-27d52ffe-7618-4a94-86ae-5d61ce857e53.jpg)

Heres the code:

.elementor-posts .elementor-post__title {
font-size: 70px;
margin-bottom: 10px;
}

.elementor-posts .elementor-post__meta-data {
font-family: 'fixturevariable';
line-height: 1.3em;
font-size: 50px;
font-weight: 700;
margin-bottom: 20px;
color: #e6008a;
}

.elementor-posts .elementor-post__read-more {
font-family: 'fixturevariable';
font-size: 30px;
font-weight: 700;
}

.elementor-posts--skin-classic .elementor-post__text {
padding-bottom: 20px;
-webkit-box-flex: 1;
-ms-flex-positive: 1;
flex-grow: 1;
}

I tried Regenerating CSS, Changing the theme and clean chache and browser history.

Do you have a solution for the problem, all help is much appreciated.

Thank you and all the best
Jón

**Comment by m1ga** [comment]
I would put that to the normal "design - customizer - custom css" section. Makes it easier to find it later on :wink:

**Reply by JonHjartarson** [reply]
Hi thanks, but it still looks the same. Do you have any idea why it doesnt work?

Best
Jón

**Reply by m1ga** [reply]
order of the scripts. Check your devtools to see which style has a higher priority. You could also add !important to you stlyes. I can only see the current page so can't help you with those styles

**Reply by JonHjartarson** [reply]
Hi hi, the  !important did the trick!!

Thank you so much!
Case closed!
